Sources/DistributedActors/Gossip/Gossiper+Shell.swift

```diff
@@ -330,7 +330,9 @@ extension GossipShell {
                     () // ignore
                 }
             }
-            context.system.cluster.events.subscribe(onClusterEventRef)
+            Task {
+                await context.system.cluster.events.subscribe(onClusterEventRef)
```

Comment:
Same issue; let's move to `nonisolated func subscribe() { Task {` so we can avoid this issue